soc/riscv/riscv-ite/it8xxx2/Kconfig.soc

```diff
@@ -11,3 +11,12 @@ config SOC_IT8XXX2
 	select ATOMIC_OPERATIONS_C
 
 endchoice
+
+config SOC_IT8XXX2_EXT_32K
+	bool "Use external 32.768 kHz clock source"
+	default n
```

Comment:
bool symbols implicitly default to n.  Zephyr style recommends skipping redundant defaults.
https://docs.zephyrproject.org/2.4.0/guides/kconfig/tips.html#redundant-defaults

Reply:
Done.